packages/rpc-api/src/__tests__/send-transaction-test.ts

```diff
@@ -249,7 +249,7 @@ describe('sendTransaction', () => {
                 innerInstructions: null,
                 logs: [],
                 returnData: null,
-                unitsConsumed: 0,
+                unitsConsumed: 0n,
```

Comment:
Why aren't these tests failing? It's because Jest 29 is stupid and doesn't match the `Error#cause` deeply enough.